src/App.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import LogPages from './component/LogPages/LogPages';
+import ContactTable from './component/ContactTable';
+import ResetPwdSuccess from './component/pages/SuccessNotification/ResetPwdSuccess/ResetPwdSuccess';
+import SignupSuccess from './component/pages/SuccessNotification/SignupSuccess/SignupSuccess'
+
+export default function App() {
+  return (
+    <>
```

Comment:
Is <></>  necessity? or replacing with a <div></div> would be more meaningful?

Reply:
Yes. It is meaningful.
It is an easy way to use react fragment to avoid empty div on browser inspector.